.circleci/config.yml

```diff
@@ -27,6 +27,12 @@ jobs:
           name: Install chromium
           command: pyppeteer-install
 
+      - run:
+          name: Install texlive
+          command: |
+            bash scripts/install_latex.sh
+            export PATH="/tmp/texlive/bin/x86_64-linux:$PATH"
```

Comment:
@choldgraf do you know how the update the environmental variable `PATH` in CircleCI?

Reply:
looks like it is something like this: https://discuss.circleci.com/t/how-to-add-a-path-to-path-in-circle-2-0/11554/4